instrumentation/hikaricp-3.0/library/src/main/java/io/opentelemetry/instrumentation/hikaricp/v3_0/OpenTelemetryMetricsTracker.java

```diff
@@ -44,20 +48,26 @@ final class OpenTelemetryMetricsTracker implements IMetricsTracker {
 
   @Override
   public void recordConnectionCreatedMillis(long connectionCreatedMillis) {
-    createTime.record((double) connectionCreatedMillis, attributes);
+    double time =
+        emitStableDatabaseSemconv()
+            ? connectionCreatedMillis / MILLIS_PER_S
+            : connectionCreatedMillis;
+    createTime.record(time, attributes);
```

Comment:
Related to #12608 

I think this should also resolve [this comment](https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/12608#issuecomment-2682836593)? 👍